README.md

```diff
@@ -25,7 +25,8 @@ java -jar plugin-management-cli/target/jenkins-plugin-manager-*.jar --war /file/
 * `--available-updates`: (optional) Set to true to show if any requested plugins have newer versions available. If a Jenkins version-specific update center is available, the latest plugin version will be determined based on that update center's data.
 * `--latest`: (optional) Set to true to download the latest version of all dependencies, even if the version(s) of the requested plugin(s) are not the latest.
 * `--latest-specified`: (optional) Set to true to download latest dependencies of any plugin that is requested to have the latest version. All other plugin dependency versions are determined by the update center metadata or the plugin MANIFEST.MF.
-* `--jenkins-update-center`: (optional) Sets the main update center, which can also be set via the JENKINS_UC environment variable. If a CLI option is entered, it will override what is set in the environment variable. If not set via CLI option or environment variable, will default to https://updates.jenkins.io.
+* `--jenkins-update-center-filename` : (optional) Sets the update filename. If not set by command line option, it will default to /update-center.json
```

Comment:
I am not sure it is necessary. `--jenkins-update-center-filename` could accept a full path, similar to how the Jenkins itself is implemented

Reply:
I'm going to push the other changes while thinking about how this would be done. Do you have a suggested strategy from Jenkins itself to identify when the string supplied is a folder and when its a file? (Or a link to code of some sort.) Doing a GET with the appended name could read some json if there was a truly pathological fouled up "update center" (although that is unlikely). Attempting a GET without the appended name might read some index file depending on the web server involved.